scripts/delayed.js

```diff
@@ -37,3 +37,30 @@ if (getConsent('commerce-collection')) {
   import('./commerce-events-sdk.js');
   import('./commerce-events-collector.js');
 }
+
+// Conversion tracking
+function convert(event, extraData = {}) {
+  sampleRUM('convert', { source: event, ...extraData });
+}
+
+window.adobeDataLayer.push((dl) => {
+  dl.addEventListener('add-to-cart', ({ eventInfo }) => {
+    convert('addToCart', { skus: eventInfo?.shoppingCartContext?.items?.map((item) => item.product.sku), element: 'cart' });
```

Comment:
`skus` and `element` will be discarded. 
We only send to backend known properties. See https://github.com/adobe/helix-rum-js/blob/0bc640edd24944c77303b7103336306850012f8e/src/index.js#L72 

you can squeeze them in the `target` field

The checkpoints is also filtered on the [collector side](https://github.com/adobe/helix-rum-collector/blob/bb1189ca43682b22e863e804257cc85880bdf8fe/src/utils.mjs#L16C17-L16C34), you would need to add these checkpoints to the list

Reply:
Could you give an example of how the addToCart RUM event should look like to meet your requirements?

Reply:
Something like this
```
convert('addToCart', { 
  target: eventInfo?.shoppingCartContext?.items?.map((item) => item.product.sku).join() 
});
```

Reply:
but you still need to whitelist the `addToCart` checkpoint in the collector

Reply:
@chicharr be aware that the `convert` function results into this:

```javascript
sampleRUM('convert', { source: 'addToCart', target: eventInfo?.shoppingCartContext?.items?.map((item) => item.product.sku).join()  });
```

Does that work for you or would you rather have `addToCart` as dedicated checkpoint like this?

```javascript
sampleRUM('addToCart', { target: eventInfo?.shoppingCartContext?.items?.map((item) => item.product.sku).join() });
```